Gemfile

```diff
@@ -6,6 +6,8 @@ gem "rails"
 gem "rake", ">= 11.1"
 gem "rubocop", ">= 0.49", require: false
 gem "rack-proxy", require: false
+# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```

Comment:
Don't think we need this inside gem

Reply:
People contributing on windows cannot run 'bundle exec rake' w/o it.

Reply:
Ahh okay makes sense 👍  Double quotes please

```rb
gem "tzinfo-data", platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```